ledger/participant-integration-api/src/main/scala/platform/store/dao/LedgerDao.scala

```diff
@@ -36,6 +36,7 @@ private[platform] trait LedgerDaoTransactionsReader {
       startExclusive: Offset,
       endInclusive: Offset,
       filter: FilterRelation,
+      wildcardParties: Set[Party],
```

Comment:
How about introducing a case class for the filter + wildCardParties? These two seem to belong very much together: they are generated together, and used together. It would be easier/clearer to pass around etc.

Reply:
done